python/lsst/ctrl/mpexec/cmdLineFwk.py

```diff
@@ -734,14 +735,16 @@ def runPipeline(
                             # Do not save fields that are not set.
                             out.write(report.json(exclude_none=True, indent=2))
 
-    def _generateTaskTable(self, qgraph: QuantumGraph) -> Table:
+    def _generateTaskTable(self, qgraph: QuantumGraph, task_max_len: int = 72) -> Table:
```

Comment:
I'm uncomfortable with this `task_max_len` value for something that is sent to a log. This goes back to the question of whether this output gets piped to a file/written to json, or just scrolled past on a screen during a run. A preserved log should not be truncated, while something written to the screen should let the terminal handle any excessive width in whatever manner it sees fit.

What would happen if you didn't set this value? Astropy's table formatter shouldn't know how wide the screen is, since this is going to the log, so wouldn't it just print the whole thing and have the terminal do the wrap-around?

Reply:
Task name truncation functionality removed from this ticket.